### PR TITLE
tealdeer: add alternative symlink for fish completions

### PR DESCRIPTION
**Pull request by tranzystorekk**
Without this, fish shell doesn't automatically load completions for the `tldr` symlink.

#### Testing the changes
- I tested the changes in this PR: **YES**

cc @filiprojek

### CODE REVIEW

File: srcpkgs/tealdeer/template [hunk 1]
```diff
@@ -12,7 +12,9 @@ homepage="https://github.com/dbrgn/tealdeer"
 changelog="https://raw.githubusercontent.com/dbrgn/tealdeer/main/CHANGELOG.md"
 distfiles="https://github.com/dbrgn/tealdeer/archive/refs/tags/v${version}.tar.gz"
 checksum=2b10e141774d2a50d25a1d3ca3d911dedc0e1313366ce0a364068c7a686300d8
-alternatives="tldr:/usr/bin/tldr:/usr/bin/tealdeer"
+alternatives="
+ tldr:/usr/bin/tldr:/usr/bin/tealdeer
+ tldr.fish:/usr/share/fish/vendor_completions.d/tldr.fish:/usr/share/fish/vendor_completions.d/tealdeer.fish"
```
Comment (classabbyamp) [inline srcpkgs/tealdeer/template:17]:
```suggestion
 tldr:/usr/share/fish/vendor_completions.d/tldr.fish:/usr/share/fish/vendor_completions.d/tealdeer.fish"
```
these should be in the same group so they are synced

Comment (classabbyamp) [reply srcpkgs/tealdeer/template:17]: also we should be able to remove the `conflicts` for `tealdeer` on `tldr`

File: srcpkgs/tldr/template
Comment (classabbyamp) [inline srcpkgs/tldr/template:1]: revbump this but otherwise lgtm
